ftrack_sftp_accessor/sftp.py

```diff
@@ -255,4 +262,6 @@ def make_container(self, resource_identifier, recursive=True):
     def get_url(self, resource_identifier=None):
         """Return url for *resource_identifier*."""
 
+        if self._folder:
+            return f"sftp://{self._host}:{self._port}/{self._prefix}/{resource_identifier}"
```

Comment:
@iwootten **self._folder** rather then "self._prefix" I guess is the variable you want to reference here.